lang/python/README.rst

```diff
@@ -140,40 +142,34 @@ keeping all of the existing fields from the original file. The
 Validating AIRR data files
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``airr`` package can validate repertoire, rearrangement, germline set
-and genotype data files
+The ``airr`` package can validate AIRR Data Model and Rearrangement TSV files
```

Comment:
This is super nit-picky, but this sounds like "Data Model TSV files and Rearrangement TSV files"...

Reply:
Good catch. Fixed in f2cb874.